### PR TITLE
AI-26 Fixed push members bug

### PR DESCRIPTION
**Pull request by Edzelopez**

### CODE REVIEW

File: CRM/Attentively/BAO/Attentively.php [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+  static public function getAttentivelyEmail($cid) {
+    if (!$cid) {
+      return NULL;
+    }
+    $sql = "SELECT email_address FROM civicrm_attentively_member
+      WHERE contact_id = {$cid}";
+    $email = CRM_Core_DAO::singleValueQuery($sql);
+    return $email;
+  }
+
```
Comment (pradpnayak) [inline CRM/Attentively/BAO/Attentively.php:241]: Can we use single function for getAttentivelyEmail(), getKloutScore(), getMemberID?

File: CRM/Attentively/BAO/Attentively.php [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+    curl_setopt( $ch, CURLOPT_POST, TRUE);
+    curl_setopt( $ch, CURLOPT_POSTFIELDS, $post);
+    curl_setopt( $ch, CURLOPT_FOLLOWLOCATION, 1);
+    curl_setopt( $ch, CURLOPT_HEADER, 0);
+    curl_setopt( $ch, CURLOPT_RETURNTRANSFER, 1);
+    $response = curl_exec( $ch );
+    $result = get_object_vars(json_decode($response));
+    return $result;
+  }
+
```
Comment (pradpnayak) [inline CRM/Attentively/BAO/Attentively.php:177]: Can't we use as a sinlge function for pushWatchedTerms(), pullPosts(), pullMember(). I see many code duplicates.

File: CRM/Attentively/BAO/Attentively.php [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+    else {
+      $url = 'http://apidev.attentive.ly/';
+    }
+    return $url;
+  }
+  
+  static public function updateAttentivelyAuth($params) {
+    $accessToken = CRM_Utils_Array::value('access_token', $params);
+    if ($accessToken) {
+      CRM_Core_DAO::singleValueQuery("UPDATE civicrm_option_value SET value = '{$accessToken}' WHERE name = 'access_token'");
```
Comment (pradpnayak) [inline CRM/Attentively/BAO/Attentively.php:50]: I think we need to add filter for option group also.

File: CRM/Attentively/BAO/Attentively.php (hunk 2 above)
Comment (Edzelopez) [inline CRM/Attentively/BAO/Attentively.php:177]: Yes, I will create an issue for this and work on optimizing code at a lower priority for the next beta release.

File: CRM/Attentively/BAO/Attentively.php (hunk 1 above)
Comment (Edzelopez) [inline CRM/Attentively/BAO/Attentively.php:241]: Yes, I will create an issue for this and work on optimizing code at a lower priority for the next beta release.

**Pull request merged**